submissions/motunrayoIlawole/challenge1-arrays/question2.js

```diff
@@ -0,0 +1,18 @@
+function solution(nums, target) {
```

Comment:
Nice and simple. Welldone. 
Time and Space Complexity are also accurate, welldone.
If you want a challenge, can you see if there's any way you could reduce the time complexity to something lower like O(n) or O(nlogn)

Sorting or [Dictionaries](https://stackoverflow.com/a/11789703) is something i think might be of help to you